website/www/site/content/en/documentation/sdks/python-custom-multi-language-pipelines-guide.md

```diff
@@ -22,15 +22,15 @@ Apache Beam's powerful model enables the development of scalable, resilient, and
 
 With SDKs available in multiple languages (Java, Python, Golang, YAML, etc.), creating and maintaining transforms for each language becomes a challenge, particularly for IOs. Developers must navigate different APIs, address unique quirks, and manage ongoing maintenance—such as updates, new features, and documentation—while ensuring consistent behavior across SDKs. This results in redundant work, as the same functionality is implemented repeatedly for each language (M x N effort, where M is the number of SDKs and N is the number of transforms).
 
-To streamline this process, Beam’s portability framework enables the use of portable transforms that can be shared across languages. This reduces duplication, allowing developers to focus on maintaining only N transforms. Pipelines combining [portable transforms](#portable-transform) from other SDK(s) are known as [“multi-language” pipelines](../programming-guide.md#13-multi-language-pipelines-multi-language-pipelines).
+To streamline this process, Beam’s portability framework enables the use of portable transforms that can be shared across languages. This reduces duplication, allowing developers to focus on maintaining only N transforms. Pipelines combining [portable transforms](#portable-transform) from other SDK(s) are known as [“multi-language” pipelines](../../programming-guide/#13-multi-language-pipelines-multi-language-pipelines).
```

Comment:
I think this needs to be `../../programming-guide/index.html#1#multi-language-pipelines` 

one can check the staged website, see from "Summary" page of beam_PreCommit_Website_Stage_GCS run: https://github.com/apache/beam/actions/runs/14223598774?pr=34512

Reply:
Hmmm the working references I see to programming-guide use the format `.../programming-guide/#XXXX`

For example: https://github.com/apache/beam/blob/da5eb2090608cbb19f67630407cc95f90197216b/website/www/site/content/en/documentation/glossary.md?plain=1#L71

Works when displayed on the website: https://beam.apache.org/documentation/glossary/#coder

Reply:
Sorry I realize links without "index.html" works for the prod website, but not the staging website. I was using the staged website to check the links: https://apache-beam-website-pull-requests.storage.googleapis.com/34512/documentation/sdks/python-custom-multi-language-pipelines-guide/index.html

Looks good to me now